.github/workflows/pr-checks.yml

```diff
@@ -51,6 +51,9 @@ jobs:
     needs: [prettier, eslint]
     runs-on: ubuntu-latest
     steps:
+      - name: Setup kernel for react native, increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
```

Comment:
what this do?

Reply:
When I added the new unit tests, tests started failing in GitHub Actions due to [this](https://github.com/expo/expo-github-action/issues/20). So I followed the fix in this thread and it solved it :)